### ISSUE TITLE
Error in documentation

### ISSUE THREAD

**Issue by carlaopereirasanca** [dart-lang/dart-pad#1533]
Learning how to embed dartpad in a web page;
Following this guide:
https://github.com/dart-lang/dart-pad/wiki/Embedding-Guide
.
There is an error, in section "Converting code blocks to DartPad", in "Step 1: Include the script".

The example says:
<script type="text/javascript" src="https://dartpad.dev/inject_embed.dart.js"></script>

This doesn't work.
Found the correct line in the example given in
https://dartpad.dev/example/inject.html

The correct line is:
<script defer src="https://dartpad.dev/inject_embed.dart.js"></script>

Best regards! Carlos

**Comment by RedBrogdon** [comment]
Thanks for filing, @carlaopereirasanca.

@johnpryan, it looks like everywhere we're importing the script (demo, codelabs), the defer keyword is being used. Is this just a little bit of docrot? If so, I can update.

**Comment by johnpryan** [comment]
Good catch - yeah the inject script needs to use defer so that it runs after the page has finished parsing (and the code snippets are ready to be converted to DartPads)

**Comment by carlaopereirasanca** [comment]
Hello Ryan and Brogdow.
Sorry to use this space for this. I tried to direct messagen any of you, without success...
Does this thing is possible in DartPad ?
https://stackoverflow.com/questions/61259838/how-to-increase-font-size-of-dartpad-embedded-in-a-web-page
tl;dr: show dartpad with greater font size. Something link control-+ on browser.
Thanks in advance. Best regards.

**Comment by domesticmouse** [comment]
@carlaopereirasanca please raise a new issue with this text sizing Feature Request, thanks!

**Comment by johnpryan** [comment]
Added this to the wiki
